src/controller/python/chip/clusters/Command.py

```diff
@@ -43,10 +43,10 @@ class CommandPath:
 
 
 @dataclass
-class InvokeRequest(CommandPath):
```

Comment:
Was `InvokeRequest` not previously used?

Reply:
It was not, I had intentions of removing the tuple before you mentioned it and that is whhat it was for. But now just did it properly